inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/codec/ProtocolDecoder.java

```diff
@@ -18,34 +18,31 @@
 
 package org.apache.inlong.sdk.dataproxy.codec;
 
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.MessageToMessageDecoder;
 import java.nio.charset.StandardCharsets;
 
-import org.jboss.netty.buffer.ChannelBuffer;
-import org.jboss.netty.channel.Channel;
-import org.jboss.netty.channel.ChannelHandlerContext;
-import org.jboss.netty.channel.DefaultExceptionEvent;
-import org.jboss.netty.handler.codec.frame.FrameDecoder;
+import java.util.List;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ProtocolDecoder extends FrameDecoder {
+public class ProtocolDecoder extends MessageToMessageDecoder<ByteBuf> {
 
-    private static final Logger logger = LoggerFactory.getLogger(FrameDecoder.class);
+    private static final Logger logger = LoggerFactory.getLogger(ProtocolDecoder.class);
 
     @Override
-    protected Object decode(ChannelHandlerContext ctx, Channel channel,
-                            ChannelBuffer buffer) throws Exception {
-        //        if(!channel.isConnected()||channel.isReadable()){return null;}
-
+    protected void decode(ChannelHandlerContext var1,
```

Comment:
Why named the variable `var1`?

Reply:
Automatically generated, unused variables